adal/src/androidTest/java/com/microsoft/aad/adal/AcquireTokenRequestTest.java

```diff
@@ -406,19 +407,23 @@ public void testLocalSilentFailedBrokerSilentReturnErrorCannotTryWithInteractive
         cacheStore.removeAll();
     }
 
-    private ITokenCacheStore getTokenCache(final Date expiresOn, final boolean storeMRRT, final boolean storeFRT) {
+    private ITokenCacheStore getTokenCache(final Date expiresOn, final boolean storeMRRT, final boolean storeFRT, final Date extendedExpiresOn) {
         // prepare valid item in cache
         final UserInfo userInfo = new UserInfo();
         userInfo.setDisplayableId(TEST_UPN);
         userInfo.setUserId(TEST_USERID);
-
-        final String accessToken = "I am an AT";
+        final String accessToken;
+        if (extendedExpiresOn ==  null) {
+            accessToken = "I am an AT";
```

Comment:
OK. I changed it back and check the ext_expires_in of the returned resulted in the verification.